etc/airflow/requirements-elyra.txt

```diff
@@ -0,0 +1,67 @@
+# This is a comprehensive list of python dependencies that Elyra requires to execute Jupyter notebooks.
```

Comment:
Could we use the same for kfp and airflow? see `etc/kfp/requirements-elyra.txt`

Reply:
yes we probably can, lets move the requirements up a level or define a new agnostic directory for it